src/jobs/microk8s_install.yml

```diff
@@ -90,7 +64,7 @@ steps:
         sudo snap install microk8s --classic --channel << parameters.microk8s_channel >>
         if [[ -z "$(echo $PATH | grep snap)" ]] ; then export PATH=$PATH:/snap/bin ; fi
```

Comment:
do we need to keep this line?

Reply:
I'll remove the test and just execute `export PATH=$PATH:/snap/bin` to add microk8s binaries to `PATH`.